primefaces-cli/pom.xml

```diff
@@ -16,6 +16,10 @@
 
     <name>PrimeFaces CLI</name>
 
+    <properties>
+        <sonar.skip>true</sonar.skip>
```

Comment:
For all modules except primefaces(-core) and primefaces-coverage we set sonar.skip=true because we probably do not want them to get included into Sonar.